features/26-mfa/api.md

```diff
@@ -1040,26 +950,33 @@ async function login(loginID: string, password: string): Promise<User> {
 }
 
 async function beginBearerTokenFlow(error: unknown): Promise<User> {
-  // Re-raise the error if it is not MFA required error.
-  if (!isMFARequiredError(error)) {
+  // Re-raise the error if it is not authentication session error
+  const authnSession = skygear.decodeAuthenticationSession(error);
+  if (authnSession == null) {
+    throw error;
+  }
+  // Re-raise the error if the step is not MFA.
+  const { step } = authnSession;
+  if (step !== "mfa") {
     throw error;
   }
+  const { token: authn_session_token } = authnSession;
 
   // Retrieve the saved bearer token
-  let bearerToken;
+  // The token may be in cookie so it cannot be retrieved.
+  let bearer_token;
   try {
-    bearerToken = await getBearerToken();
+    bearer_token = await getBearerToken();
```

Comment:
Since the bearer token may be stored in cookie/local storage, I think it's better to let SDK manage it without exposing the token. The token itself should be meaningless to developer anyways.